mesonbuild/dependencies/__init__.py

```diff
@@ -21,7 +21,8 @@
 from .dev import GMockDependency, GTestDependency, LLVMDependency, ValgrindDependency
 from .coarrays import CoarrayDependency
 from .mpi import MPIDependency
-from .misc import (BlocksDependency, HDF5Dependency, NetCDFDependency, OpenMPDependency, Python3Dependency, ThreadDependency, PcapDependency, CupsDependency, LibWmfDependency, LibGCryptDependency, GpgmeDependency, ShadercDependency)
+from .scalapack import ScalapackDependency
+from .misc import (BlocksDependency, HDF5Dependency, MPIDependency, NetCDFDependency, OpenMPDependency, Python3Dependency, ThreadDependency, PcapDependency, CupsDependency, LibWmfDependency, LibGCryptDependency, GpgmeDependency, ShadercDependency)
```

Comment:
__[Flake8]__ redefinition of unused 'MPIDependency' from line 23 ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6228#issue-4878846))

<details>

| Rule |
| - |
| F811 |

</details>
<sub>posted by Sider</sub>